react/features/recording/components/web/RecordingLimitNotificationDescription.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+
+import React from 'react';
+
+import { translate, translateToHTML } from '../../../base/i18n';
+import { connect } from '../../../base/redux';
+
+/**
+ * The type of the React {@code Component} props of {@link RecordingLimitNotificationDescription}.
+ */
+type Props = {
+    /**
+     * The limit of time for the recording.
+     */
+    _limit: number,
+
+    /**
+     * The name of the payed app with unlimited recordings.
```

Comment:
Is it important that the application requires paiment? I suggest to use either just `appName` and `appUrl`, or `unlimitedAppName` and `unlimitedAppUrl`. If you prefer the reference to payment, it should be spelled `paid`.

Reply:
Agree! Will remove the pay stuff!

Reply:
Fixed!